.github/actions/dart_package/action.yaml

```diff
@@ -80,9 +80,3 @@ runs:
       with:
         flags: ${{ steps.split.outputs.package_name }}
         token: ${{ inputs.codecov_token }}
-
-    - uses: VeryGoodOpenSource/very_good_coverage@v2
-      with:
-        path: ${{inputs.working_directory}}/coverage/lcov.info
-        exclude: ${{inputs.coverage_excludes}}
-        min_coverage: ${{inputs.min_coverage}}
```

Comment:
Removed because not all environments test all code (iOS-specific commands are only valid on MacOS, for example)

Reply:
If we're going to remove very_good_coverage I think we should update the `codecov.yml` to specify a coverage
https://docs.codecov.com/docs/codecovyml-reference#coveragerange

Reply:
Done